.circleci/regenerate.py

```diff
@@ -49,6 +49,11 @@ def build_workflows(prefix="", upload=False, filter_branch=None, indentation=6):
                     ):
                         # the fields must match the build_docs "requires" dependency
                         fb = "/.*/"
+
+                    # Keep the Python 3.8 cu116 build for docs builds until those are migrated as well.
```

Comment:
It would be nice if the build_doc job can also be migrated to GHA.
The thing is that upload_doc job requires credentials and I do not have them, so I could not do it. Perhaps release engineering team can configure the credential?

(Also currently the doc build is performed on top of binary build, but this incurs waiting time for binary build, which is not desirable. So it would be nice if we can make build_doc job separated from binary build job, by building torchaudio by itself).

Reply:
I created a follow up ticket [T141506623](https://www.internalfb.com/intern/tasks/?t=141506623).